code/modules/mob/living/simple_animal/abnormality/waw/hatred_queen.dm

```diff
@@ -208,12 +216,10 @@
 	if(!died.ckey)
 		return FALSE
 	death_counter += 1
-	//if BREACHED, check if death_counter over the death limit
-	if(breach_max_death && (death_counter >= breach_max_death))
-		GoHysteric()
 	//if CONTAINED and not crazy
-	else if((status_flags & GODMODE) && (datum_reference?.qliphoth_meter == 2) && (death_counter > 3)) // Omagah a lot of dead people!
-		BreachEffect() // We must help them!
+	if((status_flags & GODMODE) && (datum_reference?.qliphoth_meter == 2) && (death_counter > 3)) // Omagah a lot of dead people!
```

Comment:
What are you even doing here? You've just made QoH hostile breach instead of breaching in friendly mode.